masu/parts/header.html

```diff
@@ -0,0 +1,11 @@
+<!-- wp:group {"style":{"spacing":{"padding":{"top":"max(4vw, 32px)","bottom":"max(2vw, 32px)"}}},"className":"gapless-group","layout":{"inherit":"true","type":"constrained"}} -->
+<div class="wp-block-group gapless-group" style="padding-top:max(4vw, 32px);padding-bottom:max(2vw, 32px)"><!-- wp:group {"align":"wide","style":{"spacing":{"blockGap":"2rem"}},"layout":{"type":"flex","justifyContent":"space-between"}} -->
+<div class="wp-block-group alignwide"><!-- wp:group {"style":{"spacing":{"blockGap":"2rem"}},"layout":{"type":"flex"}} -->
+<div class="wp-block-group"><!-- wp:site-logo /-->
+
+<!-- wp:site-title {"style":{"elements":{"link":{"color":{"text":"var:preset|color|black"}}}}} /--></div>
+<!-- /wp:group -->
+
+<!-- wp:navigation {"__unstableLocation":"primary","overlayBackgroundColor":"beige","overlayTextColor":"burgundy","layout":{"type":"flex","setCascadingProperties":true,"justifyContent":"right","orientation":"horizontal"},"style":{"spacing":{"margin":{"top":"0"},"blockGap":"1.41em"}}} /--></div>
```

Comment:
I think the `"__unstableLocation":"primary"` can be removed as per this comment: pNEWy-f9f-p2#comment-54757

Reply:
👍